src/spinner/index.js

```diff
@@ -5,20 +5,31 @@ This source code is licensed under the MIT license found in the
 LICENSE file in the root directory of this source tree.
 */
 // @flow
+import {
+  StyledProgressBarRoundedRoot,
+  StyledProgressBarRoundedSvg,
+  StyledProgressBarRoundedText,
+  StyledProgressBarRoundedTrackBackground,
+  StyledProgressBarRoundedTrackForeground,
+} from '../progress-bar/index.js';
 export {default as Spinner} from './spinner.js';
-export {default as SpinnerDeterminate} from './spinner-determinate.js';
```

Comment:
Ah, I missed this but it seems that this main export is now missing.

```jsx
import {SpinnerDeterminate} from 'baseui/spinner';
<SpinnerDeterminate progress={progress} />
```

doesn't work anymore.

Reply:
Ah, right! 
Updated.